composer.json

```diff
@@ -11,7 +11,7 @@
     "require": {
         "php": "^7.2 || ^8.0",
         "doctrine/orm": "^2.5",
-        "sonata-project/admin-bundle": "^3.35 || ^4",
+        "sonata-project/admin-bundle": "^3.35",
```

Comment:
Dit kan voor problemen zorgen. En dan in de versie hiervoor. Ik denk dat we die "foutieve" tag ook echt moeten deleten. Anders is er straks 1 specifieke 5.x versie die opeens Sonata Admin 4 ondersteunt.

Reply:
Met semver kun je toch nooit op die versie uitkomen? Behalve als je sonata ^4 + messages-bundle ^5 definieert. Maar in het geval dat je aleen Sonata 4 hebt en je draait een `composer require zicht/messages-bundle` zou die met een bestaande 6.x op ^6 moeten uitkomen?

Reply:
Klopt... Het is een edge case, maar het kan...